src/test/java/com/testCases/UpdateUserDetails_Tcs.java

```diff
@@ -2,34 +2,130 @@
 
 public class UpdateUserDetails_Tcs extends Utilities {
 
-    String payload = "[\n" +
-            "  {\n" +
-            "    \"id\": 1,\n" +
-            "    \"username\": \"Pulkit\",\n" +
-            "    \"firstName\": \"Nitesh\",\n" +
-            "    \"lastName\": \"Dhiman\",\n" +
-            "    \"email\": \"Nitesh@gmail.com\",\n" +
-            "    \"password\": \"Test@1234\",\n" +
-            "    \"phone\": \"99965222234\",\n" +
-            "    \"userStatus\": 0\n" +
-            "  }\n" +
-            "]\n";
-
-    @Test(priority = 1, description = "Update user list of users with given array", enabled = true)
-    public void update_UserList() {
+    String payload = "{ \"id\": 568, \"username\": \"Jimmy\", \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password123\", \"phone\": \"1234567890\", \"userStatus\": 1 }";
+
+    String invalid_PhoneNumber_Payload = "{ \"id\": 568, \"username\": \"Jimmy\", \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password123\", \"phone\": \"12345621576241761567890\", \"userStatus\": 1 }";
+
+    String invalid_PasswordLength_Payload = "{ \"id\": 568, \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password12345678987654321password123\", \"phone\": \"1234567890\", \"userStatus\": 1 }";
+
+    String missing_UsernameField = "{ \"id\": 568, \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password123\", \"phone\": \"1234567890\", \"userStatus\": 1 }";
+
+    String invalidIDPayload = "{ \"id\": 5555555555555555894743876478348743t583, \"username\": \"Jimmy\", \"firstName\": \"John\", \"lastName\": \"Doe\", \"email\": \"john.doe@example.com\", \"password\": \"password123\", \"phone\": \"1234567890\", \"userStatus\": 1 }";
+
+    String emptyPayload = "";
+    @Test(priority = 1, description = "Update an existing username with PUT request", enabled = true)
+    public void updateUser_WithPUT() {
+        AllureLogger.logToAllure("Update an existing username with PUT request");
+
+        Response response = given()
+                .spec(requestSpec)
+                .body(payload).log().body()
+                .when()
+                .put("/user/new_username");
+
+        response.then().statusCode(200);
+        System.out.println(response.asString());
+    }
+
+    @Test(priority = 2, description = "Verify that sending a PUT request for a non-existent resource.", enabled = true)
+    public void updateWith_NonExistentUser() {
+        AllureLogger.logToAllure("Verify that sending a PUT request for a non-existent resource.");
+
+        Response response = given()
+                .spec(requestSpec)
+                .body(payload).log().body()
+                .when()
+                .put("/user/Username");
+
+        response.then().statusCode(200);
```

Comment:
add assertion for response payload as well ( This is must for all 200 responses)